make/platform/Makefile.linux64

```diff
@@ -17,3 +17,5 @@
 
 # Makefile.linux64
 # Special AR and RANLIB definitions for XMT used to be here.
+
+CXXFLAGS += -frtti
```

Comment:
Why is this change here? This should not be needed. You should be getting -frtti from third-party/llvm/Makefile.include-llvm - and if you're not, we need to fix that file instead.